proto/unused.proto

```diff
@@ -2,4 +2,4 @@ syntax = "proto3";
 
 package hashicorp.waypoint.sdk;
 
-option go_package = ".;proto";
+option go_package = "./;proto";
```

Comment:
My only worry is if older protobuf-gen-go won't treat this the same, I saw that previously. We might need to separate your fix from these protobuf semantic changes.